### ISSUE TITLE
Generate iterm2 files automatically

### ISSUE THREAD

**Issue by savq** [savq/melange-nvim#24]
iterm2 stores colors using apple's `plist` format (xml pretending to be json). Melange should either have a template to generate the plist automatically (as is done with other terminal emulators) or suggest a tool that targets that format.

**Comment by henryk-cesnolovic** [comment]
do you still want to generate iterm2 color presets? Or it can be done with same approach as with [alacritty](https://github.com/savq/melange-nvim/tree/master/term/alacritty) colors? I can copy colors from alacritty files to iterm and export presets.

**Comment by savq** [comment]
Yes. I removed the iterm2 files because they were not created programmatically.

Since the plist format is so verbose, I wouldn't mind using an external program to generate the file. I experimented with `vim.json` and `plutil` [here](https://github.com/savq/melange-nvim/commit/730e1123e0f8acf3c3a6bcedc29a87e851e7d6ad), and that worked ok. I never wrote a cleaned up version because I don't use iterm2 🤷

Doing the job handling in Lua is kinda verbose tho. Adding another target to the makefile should be enough. That also obviates the need to check if `plutil` is installed.
